CardScan/Classes/DetectedSSDBox.swift

```diff
@@ -19,19 +19,24 @@ struct DetectedSSDBox {
         let XMax_ = XMax * Double(imageSize.width)
         let YMin_ = YMin * Double(imageSize.height)
         let YMax_ = YMax * Double(imageSize.height)
-
+        
```

Comment:
I feel like @kingst is going to nit these spaces

Reply:
Ok. Let me fix this real quick.

Reply:
nit: let's remove this whitespace change

Reply:
Sure. Also, I tested it locally and the labels were correct.